cpp/ql/test/library-tests/types/types/types.cpp

```diff
@@ -80,3 +80,6 @@ size_t st;
 ssize_t sst;
 ptrdiff_t pdt;
 wchar_t wct;
+
+std::int8_t i8;
```

Comment:
You'll need to declare `std::int8_t` here so the test can be extracted properly.

Reply:
@rdmarsh2 -- ack, this was a mistake; that definition isn't supposed to be there.